### DISCUSSION TITLE
Is it possible to disable the filename renaming when rebuilding files?

### DISCUSSION THREAD

**Discussion by chocmake** [Yutaka-Sawada/MultiPar#128] [Q&A] [answered]
I haven't familiarized myself with the logic behind the rename detection (so it may be a string similarity comparison rather than checksum-based comparison) but is there a way to disable forcing a rename when rebuilding files based on PAR2 file parity?

Eg: if originally a file was named `My File.zip` and then MultiPar used to create sidecar PAR2 files, then subsequently the original file renamed to `My File (some comment).zip` then upon using MultiPar it will first force the user to accept renaming the original file back to `My File.zip` before rebuilding/restoring the file with the parity data.

**Comment by Yutaka-Sawada** [comment, marked as answer]
> is there a way to disable forcing a rename when rebuilding files based on PAR2 file parity?

No, MultiPar doesn't ignore misnamed files. But, it's possible to rename internal source files, after you renamed the original source files. Because some users requested similar case ago, I made two tools for such usage.

In MultiPar's help documents, there is a page about Python script. Push F1-key to open help file. Click "Help for MultiPar" on left pane. There is an item of "Python script" under "Other dicuments" section. Read subject: "Rename included source files in PAR2 sets with GUI".

If you don't want to use (or install) Python language on your PC, you may try old command-line tool in old MultiPar packages.

When you renamed included source files' name in PAR2 files, don't use "Extra" feature of MultiPar. If you want to add more PAR2 files for renamed source files later, you must newly create whole PAR2 files for new filenames.
